apps/src/calc/token.js

```diff
@@ -39,32 +39,22 @@ Token.prototype.renderToParent = function (element, xPos, markClass) {
   var text, textLength;
 
   text = document.createElementNS(Blockly.SVG_NS, 'text');
+  
+  var tspan = document.createElementNS(Blockly.SVG_NS, 'tspan');
+  tspan.textContent = this.nonRepeated_.replace(/ /g, '\u00A0\u00A0');
```

Comment:
may be worth extracting `\u00A0` or `\u00A0\u00A0` into a named variable (`nonBreakingSpaceChar` or something)